client/testutil/driver_compatible.go

```diff
@@ -18,6 +18,12 @@ func QemuCompatible(t *testing.T) {
 	}
 }
 
+func RktCompatible(t *testing.T) {
+        if runtime.GOOS != "windows" && syscall.Geteuid() != 0 {
```

Comment:
Ah, you're right. I copied this bit from the functions above it. Both of them are incorrect then. I can fix those in another patch.